apps/daengle/src/components/search/item/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { DefaultProfile } from '@daengle/design-system/icons';
+import { Tag, Text } from '@daengle/design-system';
+
+import Image from 'next/image';
+import {
+  imageStyle,
+  textBox,
+  wrapper,
+  profileWrapper,
+  DetailsWrapper,
+  opacity,
+  tags,
+} from './index.styles';
+
+interface Props {
+  image: string;
+  name: string;
+  tag: string[];
+  onClick: () => void;
+}
+
+export function Item({ image, name = '윤일', tag, onClick }: Props) {
+  return (
+    <div css={wrapper} onClick={onClick}>
+      {image === '' ? (
```

Comment:
이렇게 작성하는 게 더 와닿을 거 같습니다!
```suggestion
      {!image ? (
```

Reply:
바로 반영하도록 하겠습니댜 !